controller/workitem.go

```diff
@@ -72,6 +72,11 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 		exp = criteria.And(exp, criteria.Equals(criteria.Field(workitem.SystemArea), criteria.Literal(string(*area))))
 		additionalQuery = append(additionalQuery, "filter[area]="+*area)
 	}
+	if ctx.FilterWorkitemstate != nil {
+		wis := ctx.FilterWorkitemstate
```

Comment:
@hectorj2f I followed the same pattern as for other filters, but you are right, no need to create variables when it is called in just two places.
I will remove this variable from other filters as well then.